app/resources/v1/location.js

```diff
@@ -0,0 +1,45 @@
+require('../../../lib/db.js')
+var Street = require('../../models/street.js')
+var logger = require('../../../lib/logger.js')()
+
+exports.get = function (req, res) {
+  var handleFindStreets = function (err, results) {
+    if (err) {
+      logger.error(err)
+      res.status(500).send('Could not find streets with locations.')
+      return
+    }
+
+    if (!results) {
+      res.status(404).send('Could not find streets with locations.')
+      return
+    }
+
+    const geojson = {
+      type: 'FeatureCollection',
+      features: []
+    }
+
+    for (let i = 0; i < results.length; i++) {
+      results[i].asJson(function (err, streetJson) {
+        if (err) {
+          logger.error(err)
+          res.status(500).send('Could not render street JSON.')
+          return
+        }
+        const { location } = streetJson.data.street
+        const feature = {
+          geometry: {
+            type: 'Point',
+            coordinates: location.latlng
```

Comment:
One of the annoying things about lat/lng points is that different formats use different ordering. Leaflet does `lat, lon` (alphabetic ordering) while _most_ other systems use `lon, lat` (x, y ordering). So to make proper GeoJSON we need to flip the location coordinates.

[More info](https://macwright.org/2015/03/23/geojson-second-bite#position) and [handy chart](https://macwright.org/lonlat/).

This raises an interesting question though. Should we flip the coordinates in our data format?

Reply:
Ah. Question then, is the `lat, lng` we get from the SearchAddress in `lat, lng` ordering or `lon, lat` since I see that before you send it back to the `GeolocateDialog` you reversed it?